lib/helpers/openfga/fga.adapter.ts

```diff
@@ -52,6 +52,10 @@ export interface OpenFgaClientConfig {
   immutableBypassSecret?: string;
 }
 
+const AUTH0_PROD_HOSTS = [Auth0FgaConfiguration.getEnvironmentConfiguration(KnownEnvironment.US).apiHost,
```

Comment:
should these constant be in this file ?  Or will it be better to move it to a separate auth0fga specific file and import from it?

Reply:
Or if this file is meant for auth0fga specifically, maybe rename it as such?

Reply:
These are just env specific configs, moved them to their own file